### PR TITLE
Revert traction pagination changes

### PR DESCRIPTION
**Pull request by seenanair**
Reverting pagination to check whether Vue-tailwind package is causing deployment errors in uat for https://github.com/sanger/traction-ui/pull/947

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,8 @@ Install the require dependencies:
     npm install --include=dev
 
 NB npm 7+ installs peer dependencies by default. In our case swrv will cause resolution failures so you will need to install with the --legacy-peer-deps flag:
-https://stackoverflow.com/questions/66239691/what-does-npm-install-legacy-peer-deps-do-exactly-when-is-it-recommended-wh
+https://stackoverflow.com/questions/66239691/what-does-npm-install test:unit
```
Comment (BenTopping) [inline README.md:34]: accidentally added?

**Comment by BenTopping** [comment]
Seems fine, probably worth deploying this branch to UAT before merging into develop to see if it fixes the issue

File: README.md (hunk 1 above)
Comment (seenanair) [reply README.md:34]: I don't remember adding this :o

File: .release-version [hunk 2]
```diff
@@ -1 +1 @@
-5.4.0
+5.5.0-uat-testing-pagination
```
Comment (BenTopping) [inline .release-version:1]: probably want to revert this back to normal versioning, same with the workflow files, can remove the added lines to them

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -5,6 +5,7 @@ on:
     branches:
       - develop
       - master
+      -
```
[the remark this answers is not in the document]
Comment (seenanair) [reply .github/workflows/ci.yml]: Done

File: .release-version (hunk 2 above)
Comment (seenanair) [reply .release-version]: Done

**Review by Sangeetha-Bheeman — APPROVED** [review]
Deployed and tested in UAT. Works fine.

**Pull request merged**